### PR TITLE
Added Project Outline and Design Document Templates and created direc…

### PR DESCRIPTION
**Pull request by isaiahoh**
- Added Project Outline and Design Document Templates
- created folder for login files

Closes: #7

Signed-off-by: Isaiah Ngou <isaiahngou2000@gmail.com>

### CODE REVIEW

File: Templates/Design_Document_Template.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+How You went about testing the project, and ensured that all components of your work is functional and there are failsafe put into place when a component does not work: This is an example of things to test for:
+
+- edge cases in algorithms
+- testing webpage on all browsers and devices
+- ensure that all links are functional
+- Validate that user information is actually sent to server
+
+## Operational considerations
+
+measure how successful our product is and if theres any issues ex. latency
```
Comment (OussamaSaoudi) [inline Templates/Design_Document_Template.md:45]: Just flesh out this section real quick then it's ready to merge

File: Templates/Design_Document_Template.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Solution
+
+This section details how you went about creating a solution for your project. This section should include high level flowchart and sequence diagram, solution description, use cases, and example request and responses between other services or other components of the overall product.
+
+## Use case Diagram
+
+## Failure considerations
+
+where are areas for possible failure and how will we deal with these errors, usually user based. Ex. if unable to reach server what will you show website users?
```
Comment (OussamaSaoudi) [inline Templates/Design_Document_Template.md:28]: Capitalize

File: Templates/Project_Outline_Template.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Project Outline
+
+This Document will be created by Project Lead and given to the individual that will be working on this project. The project outline defines what an individual will contribute to the organization. It outlines functional requirements, stretch requirements, and milestone which will facilitate the achievement of the requirements.
+
+## Overview
+
+### **About the Organization (What is Oratio)**
```
Comment (OussamaSaoudi) [inline Templates/Project_Outline_Template.md:7]: Orate-io

**Comment by OussamaSaoudi** [comment]
When you push changes, feel free to resolve conversations. Once all conversations are resolved we'll merge it.

**Review by OussamaSaoudi — APPROVED** [review]
LGTM
